spec.bs

```diff
@@ -2335,11 +2407,19 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
 
 <div algorithm=snapshot-source-snapshot-params>
   Modify the [=snapshot source snapshot params=] algorithm to return a [=source snapshot params=]
-  with an additional field:
+  with three additional fields:
 
   : [=source snapshot params/initiator fenced frame config instance=]
-  :: <var ignore>sourceDocument</var>'s [=browsing context=]'s [=browsing context/fenced frame
+  :: |sourceDocument|'s [=browsing context=]'s [=browsing context/fenced frame
      config instance=]
+
+  : [=source snapshot params/attribution reporting enabled=]
+  :: The result of determining whether |sourceDocument| is [=allowed to use=] the
+     "<code>{{PermissionPolicy/attribution-reporting}}</code>" feature
+
+  : [=source snapshot params/attribution reporting context origin=]
+  :: |sourceDocument|'s [=node/context origin=]
```

Comment:
From reading https://wicg.github.io/attribution-reporting-api/#node-context-origin, it really seems like the "context origin" is meant to be called/gathered in the context of a fenced frame's main document. But here, we're calling it on the _source document_, which means we're expecting the source document to be a fenced frame, which doesn't seem right. If you want this member to be set to the fenced frame's embedder document, I'd just grab `|sourceDocument|'s `[=Document/origin=]` directly. I'm not sure what the "context origin" is used for elsewhere.

Reply:
Seems like this is mostly resolved now because https://github.com/WICG/attribution-reporting-api/pull/1070.